### DISCUSSION TITLE
mplhep.cms.label change COM energy

### DISCUSSION THREAD

**Discussion by JulesVandenbroeck** [scikit-hep/mplhep#399] [Q&A] [answered]
Hi, I am trying to change the COM energy of the mplhep.cms.label to 13.6TeV but "com=13.6" is not recognized as an argument of mplhep.cms.label. Does there exist another way to change the COM energy on the label?

Thank you in advance

**Comment by andrzejnovak** [comment, marked as answer]
Hi @JulesVandenbroeck, thanks for the bug report. You can force overwrite the right hand side label by setting `rlabel=`
